### ISSUE TITLE
Deploy public-facing packages to pypi

### ISSUE THREAD

**Issue by canwaf** [GSS-Cogs/csvcubed#13]
We can't make users pip install -e git+https://...

todo: Need to decide what our organisation's name on pypi should be - and which email address should we sign up under?

@canwaf knows all about how to name mailboxes, so he seems like the most suitable person to push this issue forwards. @JasonHowell may be required to support this too.

**Comment by robons** [comment]
Are we there on getting them to the public pypi instance yet?

**Comment by canwaf** [comment]
Cannot publish to pypi without making some modifications, which I did on the test-pypi publish. Your fork of pydantic, @robons, cannot be referenced directly via url to a GitHub repo.

```
HTTPError: 400 Client Error: Invalid value for requires_dist. Error: Can't have direct dependency: [name]
```

I have changed the existing pydantic reference in csvcubed to pydantic proper, and published 0.1.0-rc1 on test-pypi for all four packages. I am holding off until we talk about it on the proper pypi.

**Comment by canwaf** [comment]
All published now:
* https://pypi.org/project/csvcubed/
* https://pypi.org/project/csvcubed-devtools/
* https://pypi.org/project/csvcubed-models/
* https://pypi.org/project/csvcubed-pmd/
* https://pypi.org/project/csvcubed-pydantic/

With release procedure (login required): https://collaborate2.ons.gov.uk/confluence/display/PHEM/Release+Procedure
